src/components/Form/Form.css

```diff
@@ -103,81 +56,15 @@ article {
 }
 
 .wellnews-synopsis {
-  margin-top: -0.5rem;
-  transition-duration: .5s;
-  opacity: 0%;
-  display: flex;
-  height: 350px;
-  width: 50%;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-}
-
-.go-to-news {
-  height: 40px;
-  width: 60%;
-  margin-bottom: 80px;
-  color: white;
-  background-color: #44BCFF;
-  border: none;
-  font-size: 1.25em;
+  width: 75vw;
 }
 
 .hidden {
-  opacity: 100%;
+  display: none;
```

Comment:
Good choice using `display` instead of `opacity`; this should increase accessibility for screen readers.